### ISSUE TITLE
Need for accurate 3D messure tools like Solibri free viewer

### ISSUE THREAD

**Issue by olemariussvendsen** [IfcOpenShell/IfcOpenShell#3408]
As requested by the forum, Moult wished for me to write here as a request for a tool which is needed within BlenderBim.

We who receive IFC-files are often in need to take measurements, I'll give some few examples: Room height, height of the window sill, lengt of only the glas part of the window, distance from one fasade to the neighbor buildings fasade.

To cover the need to take these measures Solibri free viewer is a good tool as we in consulting engineering are not in need to edit the architect model. Thus Revit is not in our software repository. 

One tool which is from Solibri is the ability to make a section cut by activating this tool, and just click on any surface. This makes anything behind the normal direction of the click hidden. By just holding ALT and drag the mouse cursor you adjust where the section is. Same can be done by holding ALT and using middle mouse button to scroll. The advantage is then when you click on a facade and go further inwards into the section cut, it is easy to then messure the rooms Hight.

The messure tool works in Solibri like this:
- Either you click on a vertex point, line, corner where two lines meet or a face. 
- The distance is either given then in length direction of the hypotenuse, vertical or horizontal.

Solibri can not calculate angles from two points of selection, which would have been beneficial to have.

To sum up a specific usage from Solibri:

Only window glas above 0,8 m above the floor level is considered in daylight calculation. Thus you need to make sure if the top of the window sill is meeting this criteria. Click either on the window frame face, facing upwards (or line where the glas and the frame meet) and then click on the floor face. Furthermore you need also to messure the glas area of the window meaning you need length in the up and sideways direction between the window frame..

If anything is unclear, i would be happy to do a demonstration on teams/google hanghouts.

**Comment by Moult** [comment]
Closing as we now have a good set of measurement tools which are getting increasingly more advanced thanks to @brunoperdigao ! :)
